### ISSUE TITLE
Display secret names as environment variables

### ISSUE THREAD

**Issue by efekarakus** [aws/copilot-cli#720]
## Ask
Currently, if a user runs `svc show`, under "Variables" we only display environment variables:
```sh
$ copilot svc show
....
Variables

  Name                      Environment         Value
  ECS_CLI_APP_NAME          test                api
  ECS_CLI_ENVIRONMENT_NAME  test                test
  ECS_CLI_LB_DNS            test                XXXX
  ECS_CLI_PROJECT_NAME      test                ecs-kudos
  MY_DYNAMODB_TABLE_NAME    test                ecs-kudos-test-api
  MY_QUEUE_URL              test                https://sqs.us-west-2.amazonaws.com/XXXX/SampleQueue
```
Since secrets are also injected as environment variables we should also display them under variables:
```sh
$ copilot svc show
....
Variables

  Name                      Environment         Value                      Is a secret?
  ...                                                                      no
  MY_DB_PASSWD              test                ssm/<parameter name>       yes
  OTHER_DB_PASSWD           test                secretsmanager/<ARN>       yes
```

### Why?
I want to see all the available environment variable names (including secrets) when I run `"svc show"`. 

I'm not particularly attached to the UX above, so feel free to propose something else.

**Comment by kohidave** [comment]
Should we stick with the manifest style of grouping it under a different header? `Secrets` or somethign?

**Comment by efekarakus** [comment]
Sure that sounds good. It would make it easier to display too.
```
Secrets
  Name                      Environment         Value from                     
  MY_DB_PASSWD              test                ssm/<parameter name>       
  OTHER_DB_PASSWD           test                <ARN>       
```

**Comment by efekarakus** [comment]
To achieve this functionality:

#### Spec
Each checklist requires both a code change, its own unit test, and optionally split into separate PRs.

- [x] Add new method `Secrets` to `TaskDefinition`
Should be very similar to `EnvironmentVariables` method:
https://github.com/aws/copilot-cli/blob/ce04bfb49e36616cfe0bb5d3528a98d0f6d608a7/internal/pkg/aws/ecs/ecs.go#L344-L351

- [x] Add new method `Secrets` to `Service`
Should be very similar to the `EnvVars` method:
https://github.com/aws/copilot-cli/blob/f25113a582b88b6600eef38ac6b78f560b37f678/internal/pkg/describe/service.go#L104-L114

- [x] Add new type `Secret` similar to `EnvVar`, modify `backendSvcDesc` and `webSvcDesc` to use secrets
https://github.com/aws/copilot-cli/blob/d70f8cc1eb7f920748e176cc08b06e2abd20fc77/internal/pkg/describe/backend_service.go#L169-L196

- [x] Populate the `[]Secret` slice in `Describe` methods:
https://github.com/aws/copilot-cli/blob/d70f8cc1eb7f920748e176cc08b06e2abd20fc77/internal/pkg/describe/backend_service.go#L89-L118

**Comment by bvtujo** [comment]
This is addressed by the linked PR above (#1704)
